api/src/test/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivatorComponentTest.java

```diff
@@ -155,6 +159,16 @@ private void verifySentinelData() {
         assertThat(conceptService.getConcept(159947).getUuid(), is("159947AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"));
         
         assertThat(conceptService.getAllConcepts().size(), is(440));
+
+        assertThat(getOrderFrequencyUuid("162135"), is("162135OFAAAAAAAAAAAAAAA"));
+        assertThat(getOrderFrequencyUuid("162256"), is("162256OFAAAAAAAAAAAAAAA"));
+        assertThat(getOrderFrequencyUuid("162135"), is("162135OFAAAAAAAAAAAAAAA"));
+        assertThat(getOrderFrequencyUuid("162256"), is("162256OFAAAAAAAAAAAAAAA"));
+
+        assertThat(orderService.getOrderFrequencies(false).size(), is(30));
     }
 
+    private String getOrderFrequencyUuid(String conceptId) {
```

Comment:
Is the parameter a concept id or uuid?

Reply:
It is the concept Id for the concept

Reply:
It is the concept Id for order frequency concept